fs/operations/operations.go

```diff
@@ -398,8 +398,8 @@ func Move(fdst fs.Fs, dst fs.Object, remote string, src fs.Object) (newDst fs.Ob
 	}
 	// See if we have Move available
 	if doMove := fdst.Features().Move; doMove != nil && (SameConfig(src.Fs(), fdst) || (SameRemoteType(src.Fs(), fdst) && fdst.Features().ServerSideAcrossConfigs)) {
-		// Delete destination if it exists
-		if dst != nil {
+		// Delete destination if it exists and is not the same file as src (could be same file while seemingly different if the remote is case insensitive)
+		if dst != nil && (!fdst.Features().CaseInsensitive || strings.ToLower(dst.String()) != strings.ToLower(src.String())) {
```

Comment:
I think you want `dst.Remote()`  instead of `dst.String()` same with `src`.

Reply:
Okay, this change has been committed.